def test_kasitli_hata():
      assert False, 'Bu test kasıtlı olarak başarısız!'